README.md

```diff
@@ -191,6 +191,29 @@ spec:
 
 You can use `--default-role` to set a fallback role to use when annotation is not set.
 
+### Namespace Restrictions
+
+By using the flag --namespace-restrictions you can enable a mode in which the roles that pods can assume is restricted by an annotation on the pod's namespace. This annotation should be in the form of a json array.
+
+To allow the aws-cli pod specified above to run in the default namespace your namespace would look like the following.
+
+```
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    kube2iam/allowed-roles: |
```

Comment:
Shouldn't we use something more consistent with the non namespace option e.g. `iam.amazonaws.com/allowed-roles`